### ISSUE TITLE
Mypy error importing plugin

### ISSUE THREAD

**Issue by devRMA** [DisnakeDev/disnake#317]
### Summary

I'm not able to use the mypy plugin

### Reproduction Steps

First I created a venv and installed disnake by this command
```shell
pip install git+https://github.com/DisnakeDev/disnake.git
```
And then I installed mypy
```shell
pip install mypy
```
I created the ``mypy.ini`` file with the content that the [documentation](https://docs.disnake.dev/en/latest/ext/commands/slash_commands.html#number-ranges) suggests
```
[mypy]
plugins = disnake.ext.mypy_plugin
```
And finally, I made a very simple file, just to have something for mypy to analyze
```py
# main.py
from disnake.ext import commands

bot = commands.Bot(command_prefix="...")

print(bot)
```
And when I run mypy, it doesn't find the plugin
```shell
$ mypy .
mypy.ini:2: error: Error importing plugin "disnake.ext.mypy_plugin": No module named 'disnake.ext.mypy_plugin'
Found 1 error in 1 file (errors prevented further checking)
```

### Expected Results

Expected is that mypy will find the plugin

### Actual Results

Mypy can't find the plugin

### Intents

without intent

### System Information

- Python v3.10.1-final
- disnake v2.3.0-beta
    - disnake pkg_resources: v2.3.0
- aiohttp v3.8.1
- system info: Linux 5.15.16-1-MANJARO # 1 SMP PREEMPT Thu Jan 20 13:58:19 UTC 2022

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.

**Comment by shiftinv** [comment]
Fixed by #316, thanks for the report!

side note: mypy needs the `--namespace-packages` flag to properly check types in `disnake.ext.*` (unless you just omitted that in the example)

**Comment by devRMA** [comment]
> Fixed by #316, thanks for the report!
> 
> side note: mypy needs the `--namespace-packages` flag to properly check types in `disnake.ext.*` (unless you just omitted that in the example)

OK thanks. I didn't know I had to use this flag
